### ISSUE TITLE
Vampiric Bloodwine Auction house posting incorrect fee

### ISSUE THREAD

**Issue by luckystrike85** [WarEmu/WarBugs#20194]
I posted an auction for the new item from the HexenSnacht questline, the Vampiric Bloodwine for 1000g. The fee was negligible on the auction screen but once i clicked post it took 100gold from my inventory as a deposit. I doubt the potion is going to sell for 1000g so im losing my 100g. this is my first character so 100g loss is devastating to my funds. please help.

Also That questline was legit and my group had a lot of fun deciphering the clues and getting to the end. Also the extra butcher meal as well. 
**Expected behavior and actual behavior:**
I expected the actual auction house posting to be correct, 65copper and not the 100gold it took from me.

**Steps to reproduce the problem:**
Checked with a friend to post his and his notification window showed 65 copper

**Testing Screenshots/Videos/Evidences (always needed):**

![bug potion](https://user-images.githubusercontent.com/107460638/173512078-7d4f2a05-abb5-422f-893f-fdbd1221ce46.png)

**Comment by MaxHayman** [comment]
What addon are you using?

**Comment by MaxHayman** [comment]
Yes Shinies [0.9.3](https://tools.idrinth.de/addons/shinies/download/0-9-3/) is not using the correct deposit fomula.

**Comment by luckystrike85** [comment]
Does this mean im out 100g because of it?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
